vllm/core/scheduler.py

```diff
@@ -223,7 +223,11 @@ def _schedule(self) -> SchedulerOutputs:
                         continue
 
                 # If the number of batched tokens exceeds the limit, stop.
-                new_seq_lens = seq_lens + [num_prompt_tokens]
+                num_computed_tokens = self.block_manager \
+                    .get_num_computed_tokens(waiting_seq)
+                new_seq_lens = seq_lens + [
+                    num_prompt_tokens - num_computed_tokens
```

Comment:
Should this be `max(num_prompt_tokens - num_computed_tokens, 0)`?

Reply:
E.g., block size == 16, 4 blocks are cached (and 60 tokens are prefix), and if the token size is 63, it can have negative value?